src/mc-tokens.c

```diff
@@ -133,14 +125,9 @@ DEF_TOKEN_TYPE (mc_ECCDerivedFromDataToken,
                 const _mongocrypt_buffer_t *v)
 IMPL_TOKEN_NEW (mc_ECCDerivedFromDataToken, mc_ECCToken_get (ECCToken), v)
 
-DEF_TOKEN_TYPE (
-   mc_ServerDerivedFromDataToken,
-   const mc_ServerTokenDerivationLevel1Token_t *ServerTokenDerivationToken,
-   const _mongocrypt_buffer_t *v)
-IMPL_TOKEN_NEW (
-   mc_ServerDerivedFromDataToken,
-   mc_ServerTokenDerivationLevel1Token_get (ServerTokenDerivationToken),
-   v)
+DEF_TOKEN_TYPE (mc_ServerDataEncryptionLevel1Token,
```

Comment:
Suggest reverting move to `mc_ServerDataEncryptionLevel1Token` since it is not new to v2.